CustomApps/lyrics-plus/Settings.js

```diff
@@ -66,6 +67,45 @@ const CacheButton = () => {
 	);
 };
 
+const RefreshTokenButton = ({ setTokenCallback }) => {
+	const [buttonText, setButtonText] = useState("Refresh token");
+
+	useEffect(() => {
+		if (buttonText === "Refreshing token...") {
+			Spicetify.CosmosAsync.get(`https://apic-desktop.musixmatch.com/ws/1.1/token.get?app_id=web-desktop-app-v1.0`, null, {
```

Comment:
I have a bad feeling they might break this in the future :/

Reply:
I found this on a 3 years old commit and all the MXM Desktop endpoints still works to this day so I doubt that would be the case.